src/test/java/bio/terra/common/BQTestUtils.java

```diff
@@ -76,16 +77,14 @@ public static List<Map<String, Object>> mapToList(TableResult tableResult, Strin
         tableResult.getSchema().getFields().stream()
             .filter(f -> fields.length == 0 || List.of(fields).contains(f.getName()))
             .collect(Collectors.toList());
-    tableResult
-        .getValues()
-        .forEach(
-            r ->
-                returnVal.add(
-                    fieldsToProcess.stream()
-                        .collect(
-                            Collectors.toMap(
-                                Field::getName, f -> mapFieldValueToPojo(f, r.get(f.getName()))))));
+    tableResult.getValues().forEach(r -> returnVal.add(toMap(fieldsToProcess, r)));
+    return returnVal;
+  }
 
+  // We previously used Collectors.toMap but its implementation does not permit null values.
```

Comment:
More info: https://stackoverflow.com/questions/24630963/nullpointerexception-in-collectors-tomap-with-null-entry-values